packages/core/src/fonts.css

```diff
@@ -0,0 +1,125 @@
+/* Fonts ......................................................... */
+
+/* Open Sans 300 .................................................... */
+
+/* woff2 - latin-ext */
+@font-face {
+	font-family: 'Open Sans';
+	font-style: normal;
+	font-weight: 300;
+	src: local('Open Sans Light'), local('OpenSans-Light'),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') URL for "woff" format is expected. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Afont-face-browser-compatibility)